datasets/flwr_datasets/federated_dataset.py

```diff
@@ -50,6 +50,13 @@ class FederatedDataset:
         A dictionary mapping the Dataset split (a `str`) to a `Partitioner` or an `int`
         (representing the number of IID partitions that this split should be partitioned
         into).
+    shuffle: bool
```

Comment:
We're following the NumPy docstring style, which has a space before the `:`:
```suggestion
    shuffle : bool
```

Reply:
Fixed also for other parameters